pkg/app/app_test.go

```diff
@@ -91,41 +96,46 @@ func Test_createSidecar_success(t *testing.T) {
 			subnetName:        "TEST_subnet",
 			tokenName:         "TOKEN",
 			expectedTokenName: "TOKEN",
+			chainID:           "999",
 		},
 		{
 			name:              "no token name",
 			subnetName:        "TEST_subnet",
 			tokenName:         "",
 			expectedTokenName: "TEST",
+			chainID:           "888",
 		},
 	}
 
+	ap := newTestApp(t)
```

Comment:
Should we add an AfterAll to this suite to clean up the created dir or just ignore since it is already tmp?

Reply:
Because we are creating the testdir with `t.Tempdir()`:
```
// TempDir returns a temporary directory for the test to use.
// The directory is automatically removed by Cleanup when the test and
// all its subtests complete.
```